modules/store/src/selector.ts

```diff
@@ -276,8 +504,19 @@ export function createSelectorFactory(
       return projector.apply(null, selectors);
     });
 
-    const memoizedState = defaultMemoize(function(state: any) {
-      return options.stateFn.apply(null, [state, selectors, memoizedProjector]);
+    const memoizedState = defaultMemoize(function(state: any, props: any) {
+      // createSelector works directly on state
+      // e.g. createSelector((state, props) => ...)
+      if (selectors.length === 0) {
```

Comment:
Is this needed in this PR?

Reply:
It is, this will be part of the second commit to allow `createSelector((state, props) => ...)`.
This is needed because `createSelector`  only has one argument which is the projection function (there are no selectors). Meaning the `selectors` would be empty, thus our projector wouldn't be invoked.